### ISSUE TITLE
Support for external text-based playlists

### ISSUE THREAD

**Issue by lidel** [lidel/yt-looper#239]
It would be nice to decouple playlist contents from permalink in provider-agnostic way.

Idea:
- I want to manage public playlist via vim 
- prerequisite: `#list=<URL>`  loads text file from any URL
- file format
  - the same as already used URL hash 
  - alternative notation with `&` replaced by line-break (one interval per line) should be supported to ease management of long playlists
  - it should be possible to comment each line or add padding with empty lines
  - it should be possible to export current playlist to this new format

Nice to have:
- a "Refresh playlist from source" button that repeats playlist import

**Comment by lidel** [comment]
Demo: http://yt.aergia.eu/#list=http://pastebin.com/raw/WzWaTKyL

http://pastebin.com/raw/WzWaTKyL is a text file:

```
# one gif
i=VLIBa5v.gif&t=3
# one video
v=ujAevYkuMJE
```
